README.md

```diff
@@ -172,7 +172,7 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
 
         // The credentials for the rodsadmin user that will act as a proxy
         // for all authenticated users.
-        "proxy_rodsadmin": {
+        "rodsadmin": {
```

Comment:
NFSRODS uses... 
```
    "proxy_admin_account": {
```

Reply:
Right. I made this change before I realized that.

Do we want to use the NFSRODS name?

Reply:
yes please - it still looks good and is clean/clear.

Reply:
Done.